packages/openai/src/responses/convert-to-openai-responses-input.ts

```diff
@@ -204,7 +192,6 @@ export async function convertToOpenAIResponsesInput({
                 call_id: part.toolCallId,
                 name: part.toolName,
                 arguments: JSON.stringify(part.input),
-                id,
```

Comment:
Don't include id `fc_..` on client executed `function_call_output` since this is actually the reference item id on the CALL, not the result.